docs/sources/features/explore/index.md

```diff
@@ -84,24 +84,31 @@ Along with metrics, Explore allows you to investigate your logs with the followi
 - [InfluxDB](../datasources/influxdb)
 - [Elasticsearch](../datasources/elasticsearch)
 
-### Deduping
+### Visualization Options
+
+#### Deduping
 
 Log data can be very repetitive and Explore can help by hiding duplicate log lines. There are a few different deduplication algorithms that you can use:
 
 * `exact` Exact matches are done on the whole line, except for date fields.
 * `numbers` Matches on the line after stripping out numbers (durations, IP addresses etc.).
 * `signature` The most aggressive deduping - strips all letters and numbers, and matches on the remaining whitespace and punctuation.
 
-### Timestamp and Local time
+#### Time
+
+Shows or hides the time column. This is the timestamp associated with the log line as reported from the datasource.
+
+#### Unique labels
+
+Shows or hides the unique labels column that includes only non-common labels. All common labels are displayed above.
 
-There are some other check boxes under the logging graph apart from the Deduping options.
+#### Wrap lines
 
-* Timestamp: shows/hides the Timestamp column
-* Local time: shows/hides the Local time column
+Wrap-lines option for logs, as for some of our users it’s more efficient to see one line per log message. The wrapped-line option is set as a default, unwrapped setting results in horizontal scrolling.
```

Comment:
Do you mean that this option toggles line wrapping? I would say something like 

Set this to True if you want the display to use line wrapping. Otherwise, set it to False. (Or however you set this.)